src/components/SlateEditor/plugins/SlateToolbar/ToolbarButton.jsx

```diff
@@ -37,6 +38,8 @@ const toolbarIcon = t => ({
   bold: <Bold title={t('editorToolbar.bold')} />,
   italic: <Italic title={t('editorToolbar.italic')} />,
   underlined: <Underline title={t('editorToolbar.underlined')} />,
+  sub: <ExpandMore title={t('editorToolbar.sub')} />,
+  sup: <ExpandLess title={t('editorToolbar.sup')} />,
```

Comment:
Ikkje ideelle ikon, men beste eg hadde tilgjengelig.

Reply:
på ikonsiden i designmanualen står det dette:
`Systemikonene er hentet fra Google sitt Material Design som er open source og tilgjengelig med Apache License Version 2.0. Hvis det er behov for flere eller nye systemikoner skal disse hentes her: https://material.io/icons/ og hvis det ikke finnes et passende ikon i denne pakken kan man lage nye ikoner ved å bruke disse retningslinjene: https://material.io/guidelines/`
så da kan du vel finne noen nye? 
f.eks
https://material.io/resources/icons/?search=sub&icon=subscript&style=baseline
og https://material.io/resources/icons/?search=super&icon=superscript&style=baseline
og legge de til i frontend-pacakges.

Reply:
Lurt. Eg gjør det.

Reply:
Jeg syntes de sub og sup ikonene var litt små/uklare i den størrelsen toolbaren er. Er mulig det er bare jeg som syntes det da 🤷